test/integration/edition_edit_test.rb

```diff
@@ -35,4 +31,59 @@ class EditionEditTest < IntegrationTest
     assert page.has_text?("Related external links")
     assert page.has_text?("Unpublish")
   end
+
+  context "#metadata" do
+    context "#draft" do
+      setup do
+        click_link("Metadata")
+      end
+
+      should "'Metadata' header and an update button" do
```

Comment:
This does not read as a sentence. Perhaps use something like "see the 'Metadata' header and an update button" instead.